packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -8129,6 +8129,9 @@ export type OnChangeCallback = (evt?: React.FormEvent<HTMLElement | HTMLInputEle
 // @public (undocumented)
 export type OnFocusCallback = (ev?: React.FocusEvent<HTMLElement | HTMLInputElement>, props?: IChoiceGroupOption) => void | undefined;
 
+// @public (undocumented)
+export const ONKEYDOWN_TIMEOUT_DURATION = 1000;

```

Comment:
Why does this const need to be exported? It wasn't in the original change https://github.com/OfficeDev/office-ui-fabric-react/pull/9530/commits/4c6ff35d07c804286da66afed3421ea2e587f69a